ykman-gui/py/yubikey.py

```diff
@@ -168,36 +171,43 @@ def refresh(self):
-
-            self._dev_info = {
-                    'name': dev.device_name,
-                    'version': '.'.join(str(x) for x in dev.version),
-                    'serial': dev.serial or '',
-                    'usb_enabled': [
-                        a.name for a in APPLICATION
-                        if a & dev.config.usb_enabled],
-                    'usb_supported': [
-                        a.name for a in APPLICATION
-                        if a & dev.config.usb_supported],
-                    'usb_interfaces_supported': [
-                        t.name for t in TRANSPORT
-                        if t & dev.config.usb_supported],
-                    'nfc_enabled': [
-                        a.name for a in APPLICATION
-                        if a & dev.config.nfc_enabled],
-                    'nfc_supported': [
-                        a.name for a in APPLICATION
-                        if a & dev.config.nfc_supported],
-                    'usb_interfaces_enabled': str(dev.mode).split('+'),
-                    'can_write_config': dev.can_write_config,
-                    'configuration_locked': dev.config.configuration_locked,
-                    'form_factor': dev.config.form_factor
-                }
-            return success({'dev': self._dev_info})
+        try:
+            with self._open_device() as dev:
+                if not dev:
+                    return failure('no_device')
+
+                self._dev_info = {
+                        'name': dev.device_name,
+                        'version': '.'.join(str(x) for x in dev.version),
+                        'serial': dev.serial or '',
+                        'usb_enabled': [
+                            a.name for a in APPLICATION
+                            if a & dev.config.usb_enabled],
+                        'usb_supported': [
+                            a.name for a in APPLICATION
+                            if a & dev.config.usb_supported],
+                        'usb_interfaces_supported': [
+                            t.name for t in TRANSPORT
+                            if t & dev.config.usb_supported],
+                        'nfc_enabled': [
+                            a.name for a in APPLICATION
+                            if a & dev.config.nfc_enabled],
+                        'nfc_supported': [
+                            a.name for a in APPLICATION
+                            if a & dev.config.nfc_supported],
+                        'usb_interfaces_enabled': str(dev.mode).split('+'),
+                        'can_write_config': dev.can_write_config,
+                        'configuration_locked': dev.config.configuration_locked,
+                        'form_factor': dev.config.form_factor
+                    }
+                return success({'dev': self._dev_info})
+        except FailedOpeningDeviceException:
+            if self._descriptor.mode.transports == TRANSPORT.FIDO:
+                if os_is_windows:
```

Comment:
Forgot the parentheses, this will always be truthy. :slightly_smiling_face: 

```suggestion
                if os_is_windows():
```